apps/worker/src/app/workflow/services/workflow.worker.ts

```diff
@@ -63,4 +67,14 @@ export class WorkflowWorker extends WorkflowWorkerService {
       });
     };
   }
+
+  private async checkOrganizationExist(data: IWorkflowDataDto) {
+    const { organizationId } = data;
+
+    const organization = await this.organizationRepository.findOne({ _id: organizationId });
+
+    if (!organization) {
+      throw new Error('Organization not found');
+    }
```

Comment:
i think we can just make a "return" in such cases, in this time of the flow if the org is missing it is not an "error", we could add a logo that indicates of "stopped job".